.gitignore

```diff
@@ -62,6 +62,7 @@ local.conf
 .metals/
 .vscode/
 project/metals.sbt
+null/
```

Comment:
but why? )

Reply:
IDEs on windows are buggy and create a "null" folder for the sbt cache